third_party/dml/ci/test/run_tests_data.json

```diff
@@ -10,7 +10,7 @@
         "--log_device_placement"
       ],
       "pipDeps": [
-        "numpy==1.19.3",
+        "numpy==1.18.5",
```

Comment:
Was there a problem with 1.19.3?

Reply:
No, but there are various parts from the merge that explicitly reference 1.18.5. Setup.py was updated so that the tf pip package depends on version >= 1.6 and < 1.19; rather than revert the upstream change, it seems safer to keep in sync.

Reply:
That's because 1.18.5 is the last version before 1.19.x. However the only break with python 1.19.x was the C API to build TensorFlow, and we cherry-picked the appropriate fix from TensorFlow 2.0.

I don't mind going back to 1.18.5 for our CI machines, but if we do we should probably update the readme warnings to propose 1.18.5 instead of 1.19.3 since we'll have no way to make sure that 1.19.3 isn't broken in other ways that we haven't noticed yet.

Reply:
Good point on updating the README.md. Users won't even be able to use 1.19.3. It will be incompatible with our package now.